components/single_image_view.jsx

```diff
@@ -184,6 +184,7 @@ export default class SingleImageView extends React.PureComponent {
 
             fadeInClass = 'image-fade-in';
             imageStyle = {cursor: 'pointer'};
+            imageContainerStyle = {};
```

Comment:
Are you sure this doesn't reintroduce scroll pop? When I was looking at this the CSS height and this calculated height were often different

Reply:
I've checked this again, and indeed found a slight difference on height. 
I've adjusted the `image-loaded` CSS class to match the max preview height. With that, it has the same height during during image loading (`image-preload`).
I'm pretty sure that it will not reintroduce scroll pop.